src/exporter/file.js

```diff
@@ -20,7 +20,8 @@ module.exports = (node, name, ipldResolver) => {
   function visitor (node) {
     return pull(
       pull.values(node.links),
-      paramap((link, cb) => ipldResolver.get(new CID(link.multihash), cb))
+      paramap((link, cb) => ipldResolver.get(new CID(link.multihash), cb)),
+      pull.asyncMap((result, cb) => cb(null, result.value))
```

Comment:
why is this `asyncMap`, this should be just `pull.map((result) => result.value))`